packages/boa/tests/numpy/routines.matrix.js

```diff
@@ -94,22 +88,22 @@ test('numpy.matlib.randn: return a random matrix with data from the “standard
     console.log(`${n1}`);
   }
   // TODO: matrix doesn't support operators like +/-/*.
-  t.end();
+  t.pass();
 });
 
 test('numpy.matrix: operator functions', t => {
   const x = np.array([[1, 2], [3, 4]]);
   const mb = np.asmatrix(x);
   const m1 = mb.add(20);
-  t.equal(`${m1.A1}`, '[21 22 23 24]');
+  t.is(`${m1.A1}`, '[21 22 23 24]');
   // simple test
   console.log(m1.sub(10));
   console.log(m1.mul(10));
   console.log(m1.rmul(3));
   console.log(m1.imul(3));
   console.log(m1.pow(2));
   console.log(m1.ipow(3));
-  t.end();
+  t.pass();
```

Comment:
Could the result be checked like `add`?

Reply:
We need to verify if this function could be called here, because the computation logic is implemented in CPython.